traffic_ops/testing/api/v5/cdn_locks_test.go

```diff
@@ -123,19 +123,21 @@ func TestCDNLocks(t *testing.T) {
 					ClientSession: opsUserWithLockSession,
 					RequestOpts:   client.RequestOptions{QueryParameters: url.Values{"hostName": {"cdn2-test-edge"}}},
 					RequestBody: map[string]interface{}{
-						"config_apply_time": util.TimePtr(now),
+						"config_apply_time":    util.Ptr(now),
+						"config_update_failed": util.Ptr(true),
```

Comment:
need not use `util.Ptr` on non-reference field. I actually don't think it's necessary for either of these, since a non-`nil` value marshals identical to a non-reference value.

Reply:
`ServerUpdateStatusV5` still has this as a pointer since it's an optional field.